setup.py

```diff
@@ -90,12 +90,9 @@ def run(self):
         "Topic :: Scientific/Engineering :: Artificial Intelligence"
     ],
     # Package Info
-    packages=find_packages(exclude=["build*", "test*", "benchmark*", "third_party*", "build_tools*",
-                                    "datapipes_old", "examples", "scripts"]),
+    packages=find_packages(exclude=["build*", "test*", "benchmark*", "third_party*", "examples"]),
```

Comment:
nit: We can also remove `benchmark`. Let's keep `third_party` for future development